azure-functions/src/send_grid/message_builder.rs

```diff
@@ -811,32 +905,42 @@ impl MessageBuilder {
     }
 
     /// Consumes the builder and returns the SendGrid message.
-    pub fn build(self) -> SendGridMessage {
+    pub fn finish(self) -> SendGridMessage {
         self.0
     }
 
+    // PRIVATE: Appends a `to` email address to the SendGrid message addresses.
     fn append_to(&mut self, address: EmailAddress) {
         self.append_personalization();
         self.0.personalizations[0].to.push(address);
     }
 
+    // PRIVATE: Appends a `cc` email address to the SendGrid message addresses.
     fn append_cc(&mut self, address: EmailAddress) {
         self.append_personalization();
         self.0.personalizations[0].cc.push(address);
     }
 
+    // PRIVATE: Appends a `bcc` email address to the SendGrid message addresses.
     fn append_bcc(&mut self, address: EmailAddress) {
         self.append_personalization();
         self.0.personalizations[0].bcc.push(address);
     }
 
+    // PRIVATE: Instantiate personalizations for the SendGrid message if none exists.
     fn append_personalization(&mut self) {
-        if !self.0.personalizations.is_empty() {
-            return;
+        // if !self.0.personalizations.is_empty() {
+        //     return;
+        // }
+
+        // @peterhuene, I think this more clearly demonstrates the logic. What do you think?
```

Comment:
I'm fine with the way you have it and agreed we should rename it to `initialize_personalization`.